MSMetaEnhancer/app.py

```diff
@@ -15,40 +15,44 @@
 
 class Application:
     def __init__(self, log_level='info', log_file=None):
-        self.spectra = Spectra()
+        self.data = None
         logger.setup(log_level, log_file)
 
-    def load_spectra(self, filename, file_format):
+    def load_data(self, filename, file_format):
         """
         High level method to load Spectra data from given file.
 
         :param filename: path to source spectra file
         :param file_format: format of spectra
         """
-        try:
-            getattr(self.spectra, f'load_from_{file_format}')(filename)
-        except AttributeError:
+        if file_format in ['msp']:
+            self.data = Spectra()
+        elif file_format in ['csv']:
+            self.data = DataFrame()
```

Comment:
it might be useful to already now support also tsv and xlsx, because many users will have their excel sheets I'm afraid ...

Reply:
Good idea, done in [a73f40b](https://github.com/RECETOX/MSMetaEnhancer/pull/141/commits/a73f40b359920d0fbae73971aab618ae47ebb815).